src/app/parcels/ParcelsPage.tsx

```diff
@@ -181,90 +358,221 @@ const ParcelsPage: React.FC<{}> = () => {
     const searchParams = useSearchParams();
     const parcelId = searchParams.get(parcelIdParam);
 
+    const [sortState, setSortState] = useState<SortState<ParcelsTableRow>>({ sort: false });
```

Comment:
The component react-data-table-component does not allow us to set a default sort state, even though we want to sort by packing date descending by default. Therefore, in the fetch request, if the sortState is set to {sort: false}, we employ the default sorting behaviour. (see fetchParcelTableData.ts). 
Because of this limitation, the table thinks it is unsorted initially. This means that there is the annoying side effect of, if we were to click the "Packing date" column header to sort by it, the table would think "Ah yes, I need to sort by packing date ascending" so this is what the sortState is updated to. As we were already sorting by this sort field and direction, thus there is no change in the sorting that occurs in fetchParcelTableData.ts (because this depends on sortState). Then when we click it again, the table will think "Ah yes, I need to sort by packing date descending", so the sortState updates to that, and the sort behaviour is correct from now on.

Reply:
think that's good enough for now, thanks!